submissions/phpslonuk/Popup/css/style.css

```diff
@@ -65,7 +65,8 @@ a {
 }
 
 #trigger {
-  opacity: 0;
+  -webkit-appearance: none;
+  -moz-appearance: none;
```

Comment:
For future situations after properties with vendor prefixes add the same property without vendor prefixes.)